### PR TITLE
Refactor eventToAd; Add listeners for transfer events

### LINKED ISSUE #90: Wrap then publish gives error without reload
To reproduce:

1. Wrap ad
2. Immediately go to [Edit Ad]
3. Try to publish a change

Got:

> Incorrect active wallet. Must publish with: 0x...

Expected: Success

### PR DESCRIPTION
**Pull request by mveytsman**
Closes #90 

eventToAd was getting too complex. Refactored into a normalizeAd, addAd, and updateAd. This also lets us be reactive for transfer events pretty easily.

### CODE REVIEW

File: store/index.js [hunk 1]
```diff
@@ -388,63 +410,12 @@ function toAd(i, r) {
...
-    NSFW: false,
-    forceNSFW: false,
-    wrapped: false,
-  }, ad);
-}
-
-function appendAd(state, ad) {
   if (state.accounts[ad.owner]) {
     addAdOwned.call(this, state, ad);
-  } else if (ad.owner === state.networkConfig.ketherNFTAddr) {
```
Comment (shazow) [inline store/index.js:444]: Was this removed intentionally? This was the fix to #94

File: store/index.js [hunk 2]
```diff
@@ -388,63 +410,12 @@ function toAd(i, r) {
...
-    ad.y = Number(adEvent.y);
-    ad.width = Number(adEvent.width);
-    ad.height = Number(adEvent.height);
-  }
-  if (adEvent.link !== undefined) {
-    ad.link = adEvent.link;
-    ad.image = adEvent.image;
-    ad.title = adEvent.title;
-  }
-  if (adEvent.forceNSFW === true) { // Force NSFW
```
Comment (shazow) [inline store/index.js:413]: Where is this condition checked now?

File: components/App.vue [hunk 3]
```diff
@@ -196,15 +198,34 @@ export default {
...
       }.bind(this));
 
       contract.on('Publish', function(idx, link, image, title, NSFW) {
-        this.$store.commit('addAd', {idx: idx.toNumber(), link, image, title, NSFW});
-        console.log("Publish event processed.");
+        this.$store.commit('updateAd', {idx: idx.toNumber(), toUpdate: {link, image, title, NSFW}});
+        console.log("[KetherHomepage] Publish event processed.");
+      }.bind(this));
+
+      contract.on('SetAdOwner', function(idx, _from, to) {
```
Comment (shazow) [inline components/App.vue:209]: This is going to collide with initState event-based loading which also subs to this event.

File: store/index.js (hunk 1 above)
Comment (mveytsman) [reply store/index.js:444]: Yes. InitState now get this thru Transfer events.

File: store/index.js (hunk 2 above)
Comment (mveytsman) [reply store/index.js:413]: Oops added it back to normalizeAd

File: components/App.vue (hunk 3 above)
Comment (mveytsman) [reply components/App.vue]: Fixed now.

**Pull request merged**